gateway/conf.d/apicast.conf

```diff
@@ -1,3 +1,9 @@
+# {% if http_keepalive_timeout != empty %}
```

Comment:
I thought that the original issue was referring to https://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_timeout which is configured inside the `upstream` context.